modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -923,6 +923,28 @@ The product's default behavior of using a cost-based IDP search algorithm when c
 For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
 ====
 
+.Using a subquery expression within a mutating statement has implications for its side-effect visibility
```

Comment:
Now you have added this under deprecation. You should instead add it under a new section with the new code, similar to what Jack did here: https://github.com/neo4j/docs-status-codes/pull/75/files